coalib/parsing/CliParsing.py

```diff
@@ -58,7 +58,7 @@ def parse_cli(arg_list=None,
                                   line_parser)
         else:
             if isinstance(arg_value, list):
-                arg_value = ','.join([str(val) for val in arg_value])
+                arg_value = ','.join(str(val) for val in arg_value)
```

Comment:
Just a note, this type of bug is found automatically with https://pypi.python.org/pypi/flake8-comprehensions .  Oh how I wish coala could support the flake8 plugin API : https://github.com/coala/coala/issues/4229